app/models/trainee.rb

```diff
@@ -345,6 +346,10 @@ def duplicate?
     Trainee.where(first_names: first_names, last_name: last_name, date_of_birth: date_of_birth, email: email).count > 1
   end
 
+  def academic_cycle
```

Comment:
As we don't have a foreign key for `AcademicCycle` would this be better implemented as `AcademicCycle.for(<thing>)` so all the logic for what `AcademicCycle` stuff is in is encapsulated by that class? `